packages/server/src/sdk/app/automations/execution.ts

```diff
@@ -0,0 +1,57 @@
+import { context } from "@budibase/backend-core"
+import * as triggers from "../../../automations/triggers"
+import { features } from "@budibase/pro"
+import { AutomationActionStepId, Automation } from "@budibase/types"
+import env from "../../../environment"
+import sdk from "../.."
+
+export async function trigger(
```

Comment:
Do we not already have an endpoint to trigger automations?

Reply:
We do, and I considered just pinging it from here, but thought it made more sense to add it to the SDK like the rest, and call the code directly